Kickstarter-iOS/Views/Cells/BackerDashboardEmptyStateCell.swift

```diff
@@ -0,0 +1,38 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+internal final class BackerDashboardEmptyStateCell: UITableViewCell, ValueCell {
+
+  @IBOutlet private weak var iconImageView: UIImageView!
+  @IBOutlet private weak var messageLabel: UILabel!
+
+  internal func configureWith(value: ProfileProjectsType) {
+    switch value {
+    case .backed:
+      self.messageLabel.text = Strings.profile_projects_empty_state_message()
+      self.iconImageView.tintColor = .clear
+    case .saved:
+      self.messageLabel.text = localizedString(key: "todo",
```

Comment:
could you use the real key that you will add to `en.yml` that way it will just work once strings are updated even if we aren't using the `Strings.abc` helper function